data_products.tex

```diff
@@ -122,10 +122,11 @@ \subsection{Science Images
 They have undergone instrument signature removal (\secref{ssec:isr}) and all the single frame processing steps described in \secref{ssec:single_frame_processing} which are, in summary: \gls{PSF} modeling, \gls{background} subtraction, and astrometric and photometric \gls{calibration}.
 As with \texttt{raw}s, a \texttt{visit\_image} contains processed data from a single \gls{CCD} resulting from a single \exposuretime \gls{LSSTComCam} exposure.
 As a consequence, a single \gls{LSSTComCam} exposure typically results in nine \texttt{visit\_image}s.
-The handful of exposures with fewer than nine \texttt{raw} images also have fewer than nine \texttt{visit\_images}, but there are an additional \nsfpfails \texttt{raw} that failed processing and for which there is thus no corresponding \texttt{visit\_image}. Almost all failures were due to challenges with astrometric fits or \gls{PSF} models.
+The handful of exposures with fewer than nine \texttt{raw} images also have fewer than nine \texttt{visit\_images}, but there are an additional \nsfpfails \texttt{raw} that failed processing and for which there is thus no corresponding \texttt{visit\_image}.
+Almost all failures were due to challenges with astrometric fits or \gls{PSF} models in crowded fields.
 
 In total, there are \nvisitimages \texttt{visit\_image}s in \gls{DP1}.
-Each \texttt{visit\_image}  comprises three images: the calibrated science image, a variance image, and a pixel mask, indicating, for example, bad or saturated pixels, pixels affected by cosmic rays, pixels associated with detected sources, etc.
+Each \texttt{visit\_image}  comprises three images: the calibrated science image, a variance image, and a pixel bitmask, indicating, for example, bad or saturated pixels, pixels affected by cosmic rays, pixels associated with detected sources, etc.
```

Comment:
"mask" -> "bitmask" is a concise way to dispel the common assumption that it's boolean.